src/User.js

```diff
@@ -0,0 +1,16 @@
+class User {
+  constructor(userData) {
+    this.userData = userData;
+  }
+
+  returnFristName(){
+    const firstName = this.userData.name.split(" ");
+    console.log(firstName);
+    return firstName[0];
+  }
+}
+
+
+if (typeof module !== 'undefined') {
+  module.exports = User;
```

Comment:
Let's ask about what exactly this does, maybe just for curiosity sake. Assuming this allows it to work on a webpage without breaking like the export usually would.